Signal/src/Models/OWSMessagesBubblesSizeCalculator.m

```diff
@@ -120,7 +120,7 @@ - (CGSize)sizeForSystemMessage:(TSInteraction *)interaction
     }
 
     if (!self.referenceSystemMessageCell) {
```

Comment:
nit: If this change is from my feedback I did not make myself clear. I was suggesting you don't create these reference cells as a side effect of calling the `sizeForSystemMessage` (the name of which does not imply side effects), but rather we'd continue to use the property accessor as before, and implement the getter something like this:

```
@property (nonatomic, readonly) OWSSystemMessageCell *referenceSystemMessageCell;
...
@synthesize referenceSystemMessageCell = _referenceSystemMessageCell;
- (OWSSystemMessageCell *)referenceSystemMessageCell
{
  if(_referenceSystemMessageCell == nil) {
     _referenceSystemMessageCell = [OWSSystemMessageCell new];
  }
  return _referenceSystemMessageCell;
}
```

No biggie, but I think it's more resilient to future assumptions if properties never return nil (when reasonable).